crypto/x509/x_attrib.c

```diff
@@ -91,7 +91,7 @@ static int asn1_integer_print_bio(BIO *bio, const ASN1_INTEGER *num)
     return result;
 }
 
-static int print_oid (BIO *out, ASN1_OBJECT *oid) {
+static int print_oid(BIO *out, const ASN1_OBJECT *oid) {
```

Comment:
When you're fixing things here, could you please fix the leak of `num_bn` on line 87 in the previous function?